### DISCUSSION TITLE
Hero 7 Black - USB power trigger works but unit powers on randomly

### DISCUSSION THREAD

**Discussion by snake2332** [gopro/labs#1023] [Q&A]
I have disabled all extra features on this GoPro (wifi, GPS, etc) but no matter what, it just powers on by itself at a seemingly random amount of minutes later, which uses all the battery but also aborts the USB power-trigger "dashcam mode". 

I actually don't want it to record, so I took the !S out of the QR code recently. I just want it to use as a 1080p60 rear view mirror in my vehicle, since none of the rearview camera mirrors do 60fps. They are all 25fps which I hate with a passion - personal issue I know, haha!

I have done the !reset and a factory reset to make sure there isn't some sort of motion or sound trigger that would power on the unit. Does anyone know if there is a fix for this issue or something I'm missing? 

Does the GoPro 8 actually work reliably with USB power-trigger? If so, this 7 is going up on the auction block. 

Thanks!

**Comment by snake2332** [comment]
I had the auto-power-off set to Never and changed it to 5 minutes - doesn't matter. When the GoPro decides it wants to power on for no reason, it still sits there until the battery dies and has lost the USB power trigger boot sequence.

**Comment by dnewman-gpsw** [comment]
This feature never worked reliably on HERO7.

Also for you non-recording purposes you are likely using the wrong camera and the wrong script.  Use a HERO8 or better, with WAKE=2 enabled. The camera will boot every time you provide power.  However I expect you are using HDMI, and HERO7 was the last to have that integrated. So there might be no easy solution.

**Reply by snake2332** [reply]
Thanks, that convinces me the 7 is no good for my needs. I tried the visual basic method initially since it isn't obvious that those won't work with the 7, then was getting !Lbt messages on boot-up, then fell back on the more basic script.

Yes, using a 25' HDMI cable to get from rear of vehicle to Feelworld monitor on dash. Hero 8+ requires the Media Mod, so I was shopping bundles on eBay last night. Not too expensive - hopefully I can get $100 for this 7. I will do just the wake=2 command like you mention since I just want it to boot up and go to the default mode while will be Video set to 1080p60 Linear.

Cheers!

**Reply by dnewman-gpsw** [reply]
for HERO8 the commands will be !MHMDI=1!MWAKE=2 that is all you will need.  Provide power and the camera will turn on and HDMI will be clean.

**Reply by snake2332** [reply]
Funnily enough, I can't get it to fail anymore. It's worked correctly for the last 7 hours, with me testing it 5 or 6 times in that time-span. No clue why, maybe because the battery is under 35% but I doubt that. Still, I don't trust it. I'm waiting on the HDMI cable to arrive and don't even have the monitor yet, so I'm just vetting it out still. It would be a bit annoying to have to keep the QR code stored on my phone then flash it in front of the GoPro every time I notice that it's messed up. Wife is money-aggro'd already though so I don't want to spend more on this stuff, so I'll keep trying the 7 and push off the expense of a version 8 for a couple weeks at least.

**Reply by snake2332** [reply]
Going on 2 days now of it working as it should. The last change I made was to enable screen-lock.

I got the microHDMI cable and hooked it up. The GoPro will not power off if the monitor is on, but if you turn off the monitor then unplug the USB power, it will shut down.
